server/boot/challenge.js

```diff
@@ -24,10 +24,12 @@ function buildUserUpdate(
 
   if (alreadyCompleted) {
     // add data from old challenge
-    finalChallenge = {
+      const count = oldChallenge.submitCount ? oldChallenge.submitCount + 1 : 0;
```

Comment:
Let's keep this `numOfAttempts` as the addition to the schema is defined. This is also much more descriptive than `submitCount`.

Reply:
Since this variable is used in two places, it can be moved up in scope and used in this condition and the one below.